ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1728,32 +1728,18 @@ object JdbcLedgerDao {
 
     override protected[JdbcLedgerDao] val SQL_SELECT_CONTRACT: String =
       s"""
-         |select
-         |  cd.id,
-         |  cd.contract,
-         |  c.transaction_id,
-         |  c.create_event_id,
-         |  c.workflow_id,
-         |  c.key,
-         |  le.effective_at,
-         |  string_agg(distinct sigs.signatory, '$PARTY_SEPARATOR') as signatories,
-         |  string_agg(distinct obs.observer, '$PARTY_SEPARATOR') as observers
+         |select cd.contract
```

Comment:
I guess without the string aggregation the query could even be moved out of the database specific traits?

Reply:
For sure, I just wanted to solve one issue at a time and keep the PR as simple as possible.

Reply:
Tracked by https://github.com/digital-asset/daml/issues/4544

Reply:
Addressed in https://github.com/digital-asset/daml/pull/4545